PyFunceble/cli/scripts/iana.py

```diff
@@ -190,7 +190,7 @@ def set_destination(self, value: str) -> "IanaDBGenerator":
 
         return self
 
-    def get_referer_from_extension(self, extension: str) -> Optional[str]:
+    def get_referrer_from_extension(self, extension: str) -> Optional[str]:
```

Comment:
I hope for you that the `iana-pyfunceble` CLI is working after this change 😄

Reply:
Admitted I haven't tested; but you gave it the :ok: :arrow_double_up: so I expected it to be _all_ files 

so I went for the global find-replace :smirk: 

I'll mark it unread, and change it to it's misspelled counterpart tomorrow

Reply:
You got a few other fixes at t he same price :feet:

Reply:
I think you have a typo in you coveralls

https://coveralls.io/builds/36631621, otherwise I can't spot why this build should be falling, and a pointer would be prefered